pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -196,6 +197,7 @@ class Build {
 												context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
 												context.string(name: 'JDK_REPO', value: jdkRepo),
 												context.string(name: 'JDK_BRANCH', value: jdkBranch),
+												context.string(name: 'OPENJ9_BRANCH', value: openj9Branch),
```

Comment:
why context as opposed to text?

Reply:
No particular reason. All existing values are using `context.string`.

Reply:
Text provides a big, multiline text box similar to the targetConfigurations box in the openjdk-pipeline jobs. String provides a much smaller, single line box. Seeing as it's a branch name being passed here, I would say that string is suitable